cookbook/bundles/configuration.rst

```diff
@@ -294,8 +294,8 @@ The ``config:dump-reference`` command dumps the default configuration of a
 bundle in the console using the Yaml format.
 
 As long as your bundle's configuration is located in the standard location
-(``YourBundle\DependencyInjection\Configuration``) and does not require
-arguments to be passed to the constructor it will work automatically. If you
+(``YourBundle\DependencyInjection\Configuration``) and does not extend
+its constructor it will work automatically. If you
```

Comment:
It works either way, I just used `extend` because this still works if the parent `Configuration` class has a __construct method itself.

Still I agree that "does not have a constructor" makes the meaning very clear. Waiting for any other feedback before changing the PR :)